dvc/api.py

```diff
@@ -91,6 +93,24 @@ def read(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
         return fd.read()
 
 
+def get_params(stage: str, flatten: bool = False) -> Dict:
+    """Returns all the params associated with the given `stage`.
+
+    Args:
+        stage (str): Name of the dvc.yaml stage to get params from.
+        flatten (bool, optional): Whether to return flattened dict
+        or not. Defaults False.
+        If True, {"foo.bar": 1} would be returned, instead of
+        {"foo": {"bar": 1}}
+    """
+    repo = Repo()
+    params = {}
+    for dep in repo.stage.load_one(name=stage).deps:
```

Comment:
This won't work if the working directory is in a different location than the `dvc.yaml` file. This could happen if the script was invoked by `repro/exp-run` and the stage has `wdir` set, or the user is manually invoking them through different directory. 

Do we want to extend the API to pass the path as well?

Reply:
> Do we want to extend the API to pass the path as well?

No strong opinion. I originally wanted to keep API as simple as possible.

Do you think the different workdir would be a common scenario?